frontends/election-manager/src/app.test.tsx

```diff
@@ -339,8 +339,10 @@ test('L&A (logic and accuracy) flow', async () => {
     exact: false,
   });
   expect(printer.print).toHaveBeenCalledTimes(2);
```

Comment:
Now that we're also printing tally reports, I _think_ we expect one more print call here. If we move this check after the `await waitFor` below, we should be able to check for 3 printer calls